test/Dialect/EmitC/ops.mlir

```diff
@@ -1,5 +1,8 @@
 // RUN: emitc-opt -allow-unregistered-dialect -verify-diagnostics %s | FileCheck %s
 
+"emitc.include" (){include = "<test.h>"} : () -> ()
+emitc.include "<test.h>"
```

Comment:
nit: The angel brackets are irritating, as the type of include is handled by an attribute

```suggestion
"emitc.include" (){include = "test.h"} : () -> ()
emitc.include "test.h"
```

Reply:
Good catch! Missed to update this test while introducing the unit attribute. Fixed :)